peps/pep-0756.rst

```diff
@@ -241,31 +215,29 @@ See ``PyUnicode_Export()`` for the available formats.
 UTF-8 format
 ------------
 
-CPython 3.14 doesn't use the UTF-8 format internally. The format is
-provided for compatibility with PyPy which uses UTF-8 natively for
-strings. However, in CPython, the encoded UTF-8 string is cached which
-makes it convenient to be exported.
+CPython 3.14 doesn't use the UTF-8 format internally and doesn't support
+exporting a string as UTF-8. The ``PyUnicode_AsUTF8AndSize()`` function
+can be used instead.
+
+The ``PyUnicode_FORMAT_UTF8`` format is provided for compatibility with
+PyPy which uses UTF-8 natively for strings.
```

Comment:
```suggestion
The ``PyUnicode_FORMAT_UTF8`` format is provided for compatibility with
alternate implementations which may use UTF-8 natively for strings.
```

I believe MicroPython also uses UTF-8 internally, but the point is to allow an API that is adaptable and can be consistent for other implementations, so we don't really need to specify any particular implementation.

Providing a UTF-16 value for the same purpose probably wouldn't hurt. Nobody _has_ to implement it, and if a caller was going to receive it and then convert anyway then they should not specify the type and call one of the converting functions.